Consolidated <bindings> element documentation (#10099)
dotnet/docs: docs/framework/configure-apps/file-schema/wcf/bindings.md

---
title: "&lt;bindings&gt;"
ms.date: "01/22/2018"
ms.assetid: b62cd369-5409-4030-8490-9759a462dd3a
---
# &lt;bindings&gt;

> ---

Reviewer: you have an extra yml section here

  
## System-provided bindings
 
 System-provided bindings hide the complexity of the WCF messaging stack. Applications using system-provided bindings do not require full control over the stack. The attributes exposed on each system-provided binding are the ones most appropriate for the usage scenario the binding addresses.  
  
 The configuration section for each system-provided binding can define several configurations used to configure the binding. Each configuration is identified by a unique name.  
  

>  It is not possible to add elements or attributes to a system-provided binding. To do so, you should implement a custom binding as described in the [Custom Bindings](#custom-bindings)section. It is possible to define a custom binding that mimics a system-provided binding perfectly and adds a few settings the user application wants to have control over.  

Reviewer: ```suggestion
 It isn't possible to add elements or attributes to a system-provided binding. To do so, you should implement a custom binding as described in the [Custom bindings](#custom-bindings) section. It's possible to define a custom binding that
  mimics a system-provided binding perfectly and adds a few settings the user application wants to have control over.  
```

 Custom bindings are identified by their `name` attribute. For more information on custom bindings, see [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md).  
  
## See also

- <xref:System.ServiceModel.Configuration.BindingsSection?displayProperty=nameWithType>  
- <xref:System.ServiceModel.Channels.Binding?displayProperty=nameWithType>  
- <xref:System.ServiceModel.Channels.BindingElement?displayProperty=nameWithType>  
- [Bindings](../../../../../docs/framework/wcf/bindings.md)  
- [Custom Bindings](../../../../../docs/framework/wcf/extending/custom-bindings.md)  
- [\<customBinding>](../../../../../docs/framework/configure-apps/file-schema/wcf/custombinding.md)  

> - [\<binding>](../../../../../docs/framework/misc/binding.md)

Reviewer: you should see if you have more incoming links like this to the old topic and update them to point to this new one. This one needs to be removed though.
